opencti-platform/opencti-graphql/src/utils/syntax.js

```diff
@@ -168,6 +169,9 @@ export const checkObservableSyntax = (observableType, observableData) => {
     case C.ENTITY_MAC_ADDR:
       if (!macAddrChecker.test(observableData.value)) return 'Valid MAC address';
       break;
+    case C.ENTITY_SOFTWARE:
+      if (!observableData.name && !cpeChecker.test(observableData.cpe) && !observableData.swid) return 'Valid Software attributes';
```

Comment:
We could extract this condition to a constant `areValidSoftwareAttributes` to explicit it